esprima.js

```diff
@@ -1274,27 +1281,13 @@
 
                     default:
                         if (isOctalDigit(ch)) {
```

Comment:
@mathiasbynens Can you elaborate more? Why is that related to template literals where octals has never been defined?